server/BraveAlerterConfigurator.js

```diff
@@ -185,6 +208,7 @@ class BraveAlerterConfigurator {
         break
 
       case CHATBOT_STATE.COMPLETED:
+      case CHATBOT_STATE.RESET:
         returnMessage = null
```

Comment:
This one seems OK as `null` because the OtherResponderPhones probably wouldn't have been expecting a text anyways.